src/fixtures/stubbed_saved_object_index_pattern.ts

```diff
@@ -44,5 +44,12 @@ export function stubbedSavedObjectIndexPattern(id: string | null = null) {
       title: 'title',
     },
     version: '2',
+    reference: [
+      {
+        id: 'id',
+        name: 'name',
+        type: 'data-source',
+      },
+    ],
```

Comment:
shall we add a parameter to this function to control if we want to add reference in the doc or not? we should test both multi-source enable and disable cases.

Reply:
make sense